addons/event_sale/tests/common.py

```diff
@@ -44,3 +46,66 @@ def setUpClass(cls):
             'date_end': fields.Datetime.to_string(datetime.today() + timedelta(days=15)),
             'date_tz': 'Europe/Brussels',
         })
+
+
+class TestEventSaleTicketWithPriceListCommon(TestEventSaleCommon):
+    @classmethod
+    def setUpClass(cls):
+        super(TestEventSaleTicketWithPriceListCommon, cls).setUpClass()
+        ResCurrency = cls.env['res.currency'].with_context({'active_test': False})
+        cls.usd_currency = ResCurrency.search([('name', '=', 'USD')])
+        cls.eur_currency = ResCurrency.search([('name', '=', 'EUR')])
+        cls.company_currency = cls.env.company.currency_id
+        cls.other_currency = cls.eur_currency if cls.company_currency != cls.eur_currency else cls.usd_currency
+        cls.precision_delta = cls.other_currency.rounding
+        cls.price_list = cls.env['product.pricelist'].create({
+            'name': 'Sale pricelist',
+            'discount_policy': 'with_discount',
+        })
+        cls.tax = cls.env['account.tax'].create({
+            'name': "Tax 10",
+            'amount': 10,
+        })
+        cls.event_product.write({'taxes_id': [Command.set([cls.tax.id])], })
+        cls.event_product_templ.write({'taxes_id': [Command.set([cls.tax.id])], })
+        cls.now = datetime.now()
+        cls.rule_type_to_field = {  # define which field should be written for a given rule_type
+            'fixed': 'fixed_price',
+            'percentage': 'percent_price',
+            'formula': 'price_discount',
+        }
+        cls.product_or_template = cls.event_product
+
+    def configure_pricelist(self,
+                            rule_type, rule_amount, min_qty,
+                            rule_based_on=None,
+                            price_list_currency=None):
```

Comment:
it doesn't bother me much, but we usually would do something like this instead
```suggestion
    def configure_pricelist(self, rule_type, rule_amount, min_qty,
                            rule_based_on=None, price_list_currency=None):
```

Reply:
yes that’s 2 lines less